evm/__init__.py

```diff
@@ -21,8 +21,8 @@
 #  Setup TRACE level logging.
 #
 logging.addLevelName(TRACE_LEVEL_NUM, 'TRACE')
-logging.TRACE = TRACE_LEVEL_NUM
-logging.Logger.trace = trace
+logging.TRACE = TRACE_LEVEL_NUM  # type: ignore
+logging.Logger.trace = trace  # type: ignore
```

Comment:
Why do we have to ignore those?

Reply:
Because these properties do not exist and we are making them up out of thin air. I'll put a comment on it.